src/components/Projects.tsx

```diff
@@ -2,33 +2,97 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { makeStyles } from '@material-ui/core/styles';
 import { Container, Box, Paper } from '@material-ui/core';
```

Comment:
All imports can go in one line

`import { Card, Container, Box, CardActions, etc } from '@material-ui/core`

Reply:
fixed